lambda-kinesis-lambda/README.md

```diff
@@ -0,0 +1,70 @@
+# A demonstration of serverless producer-consumer use case using AWS Lambda and AWS Kinesis Data Stream
+
+This pattern creates an AWS Kinesis Data Stream, a stream consumer, and an AWS Lambda function. When data is added to the stream, the Lambda function is invoked.
+
+Learn more about this pattern at Serverless Land Patterns: [https://serverlessland.com/patterns/kinesis-to-lambda](https://serverlessland.com/patterns/kinesis-to-lambda)
```

Comment:
right endpoint will be same as this directory - https://serverlessland.com/patterns/lambda-kinesis-lambda